jobsub/examples/pALPIDEfs/steering-templates_pALPIDEfs_7/hotpixel-tmp.xml

```diff
@@ -26,7 +26,7 @@
    <global>
       <parameter name="LCIOInputFiles"> @LcioPath@/@FilePrefix@-converter.slcio </parameter>
       <parameter name="GearXMLFile" value="@GearFilePath@/@GearFile@"/>
-      <parameter name="MaxRecordNumber" value="@nEventsForHotpixel@"/>
+      <parameter name="MaxRecordNumber" value="@MaxRecordNumber@"/>
```

Comment:
@mkofarag 
MaxRecordNumber is set inside the config file. 
I've changed those because there was some buggy stuff on-going when I set MaxRecordNumber = NoOfEvents